README.md

```diff
@@ -15,3 +15,18 @@ Go to the project root directory.
 git config user.name "John Doe"
 git config user.email "john.doe@mail.com"
 ```
+
+2. Install Poetry package manager (a system requirement for this project).
+```bash
+pip install poetry
+```
```

Comment:
I would suggest to not provide an exact way of how to install poetry, because:
1. Some developers may already have poetry installed.
2. Some developers would prefer to install poetry in another way.

We actually are not interested so much in HOW the user will install poetry and we want give him a choice. So I think we can just mention that this project requires Poetry in **System Dependencies** section, and leave a link to the official poetry installation guide.

Reply:
This **System Requirements** section will be also helpful after some time, when our project will grow and we will have more system-related settings required for it